CONTRIBUTING.md

```diff
@@ -47,47 +47,50 @@ First, ensure that [Yarn](https://yarnpkg.com/) and
 
 ## The demo project
 
-The [demo project](demo) was initially forked from
-[dvc-checkpoints-mnist](https://github.com/iterative/dvc-checkpoints-mnist/tree/make_checkpoint),
-and is provided as a lightweight, convenient testbed to try the extension out
-with. It is not an exhaustive showcase of DVC's features, so testers are
-encouraged to try other DVC repositories -- especially real-world cases!
+The [demo project](demo) was initially forked from [dvc-checkpoints-mnist], and
```

Comment:
let's remove this reference, it doesn't matter at all now I think

Reply:
Agreed. Esp. since https://github.com/iterative/dvc-checkpoints-mnist has no LICENSE (shoult it?)

Removed.